### PR TITLE
Fix our artisan commands

### PR DESCRIPTION
**Pull request by doshitan**
Also mark the artisan script as executable.

Issue #957 

@krues8dr @sethetter

### CODE REVIEW

**Comment by krues8dr** [comment]
@doshitan I think this is failing in Travis because Laravel 5.1 changed how arguments are read?  https://laravel.com/docs/5.1/artisan#writing-commands

**Comment by krues8dr** [comment]
It looks like those are using the 5.0 style, iirc

File: server/app/Console/Commands/DatabaseRebuild.php [hunk 1]
```diff
@@ -3,28 +3,32 @@
...
 class DatabaseRebuild extends Command
 {
     /**
-     * The console command name.
+     * The name and signature of the console command.
      *
      * @var string
      */
-    protected $signature = 'db:rebuild {--database=mysql}';
+    protected $signature = 'db:rebuild {--database=}';
```
Comment (sethetter) [inline server/app/Console/Commands/DatabaseRebuild.php]: Does this effectively remove the default?

Comment (sethetter) [inline server/app/Console/Commands/DatabaseRebuild.php]: Looks like it does. This change breaks the `make db-reset` command. Fixing that now.

Comment (sethetter) [inline server/app/Console/Commands/DatabaseRebuild.php]: Fixed in 4dc9b8d

Comment (doshitan) [inline server/app/Console/Commands/DatabaseRebuild.php]: It does remove the default as defaulting to a database called `mysql` seemed like a bad idea, but I see now that it doesn't specify the actual database to use, but rather which application database connection to use, so it's just badly named. I see that Laravel's commands use the same terminology though, so I guess we'll keep it. I'll add the default back in.

**Comment by sethetter** [comment]
:+1: on this one. As long as @krues8dr has no objection, merge away @doshitan!

**Comment by krues8dr** [comment]
:+1:

**Pull request merged**